models/classes/search/ResultSetResponseNormalizer.php

```diff
@@ -70,18 +76,39 @@ public function normalize(SearchQuery $searchQuery, ResultSet $resultSet, string
                     $content['label'] = __('Access Denied');
                 }
 
+                $resource = $this->getResource($content['id']);
```

Comment:
Minor one, but WDYT  - with fallback we sould be  safier 
```suggestion
                $resource = $this->getResource($content['id']??'');
```